<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ButonSecim">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Seçimleri Al</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelResult">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>181</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>Seçimler</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="CbSinema">
       <property name="text">
        <string>Sinema</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="CbKitap">
       <property name="text">
        <string>Kitap Okumak</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="CbSpor">
       <property name="text">
        <string>Spor</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
